.github/workflows/rust.yml

```diff
@@ -21,13 +21,21 @@ jobs:
 
     strategy:
       fail-fast: false
+      matrix:
+        rust:
+          - 1.75.0
+          - stable
 
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
 
       - name: Install Rust toolchain
-        uses: actions-rust-lang/setup-rust-toolchain@v1
+        uses: actions-rs/toolchain@v1
```

Comment:
I see you switched the actions you use to `actions-rs/toolchain`.
This one doesn't seem to be maintained anymore, so I'm curious about that. Doesn't it work with `actions-rust-lang/setup-rust-toolchain` before the change?
https://github.com/actions-rs/toolchain/issues/216

Reply:
Hmmm, ok! I had changed it because I thought the [option to configure the `profile` as `minimal`](https://github.com/sugyan/atrium/pull/233/files/d751f6a53529ad30886179e3c1551338e1454649#diff-73e17259d77e5fbef83b2bdbbe4dc40a912f807472287f7f45b77e0cbf78792dR36) seemed particularly appealing. But I don't think it's that big of a deal. I'll change it back.
Thanks for taking note of it!